src/components/wallets/WalletAccordion.tsx

```diff
@@ -0,0 +1,357 @@
+import { useState } from 'react'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import styled, { css } from 'styled-components'
+import { DateTime } from 'luxon'
+import { gql } from '@apollo/client'
+
+import { theme } from '~/styles'
+import {
+  Avatar,
+  Button,
+  Icon,
+  Skeleton,
+  Status,
+  StatusEnum,
+  Tooltip,
+  Typography,
+} from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { WalletDetailsFragment, WalletStatusEnum } from '~/generated/graphql'
+
+gql`
+  fragment WalletDetails on Wallet {
+    id
+    balance
```

Comment:
Not used

Reply:
It should have ! Updated the template

We also realised that we're missing an attribute on the backend side
@jdenquin will add consumed_balance soon and it will be used in this template

Reply:
Added a note about that in the template so I can handle it later